src/tests/regressionTests.test.tsx

```diff
@@ -847,4 +847,48 @@ describe("regression tests", () => {
     fireEvent.click(screen.getByText(/code/i));
     expect(h.state.currentItemFontFamily).toEqual(3); // Cascadia
   });
+
+  it("shows context menu for canvas", () => {
+    fireEvent.change(document.querySelector(".dropdown-select__language")!, {
+      target: { value: "en" },
+    });
```

Comment:
`en` should be the default for tests, or did it not work without explicitly setting it? If so, I'd do it  programmatically before each test, instead of via UI.

Reply:
There is one test which sets it to `de-DE` which is why i needed to reset it. We could do as you suggested.

Reply:
Right, I think that was one of mine, and I forgot to do reset it (viva la shared state...).